int32 big_node
float64 ld
float64 distance
float64[] s_x
float64[] s_y
float64[] s_a
float64[] s_d
float64 tm_x
float64 tm_y
